charts/free5gc/charts/free5gc-smf/values.yaml

```diff
@@ -57,7 +57,7 @@ smf:
   replicaCount: 1
   image:
     name: free5gc/smf
-    tag: v3.4.3
+    tag: latest
```

Comment:
Please specify the digest, instead of using latest tag.
![image](https://github.com/user-attachments/assets/4343a8c4-8064-4e12-8d83-61ad1ebdde96)